--- origsrc/cairo-1.6.4/configure.in	2008-04-11 16:33:05.000000000 -0500
+++ src/cairo-1.6.4/configure.in	2008-05-13 21:42:38.187500000 -0500
@@ -26,24 +26,24 @@
 # libtool shared library version
 
 # Increment if the interface has additions, changes, removals.
-LT_CURRENT=19
+CAIRO_LT_CURRENT=19
 
 # Increment any time the source changes; set to
 # 0 if you increment CURRENT
-LT_REVISION=5
+CAIRO_LT_REVISION=5
 
 # Increment if any interfaces have been added; set to 0
 # if any interfaces have been removed. removal has
 # precedence over adding, so set to 0 if both happened.
-LT_AGE=17
+CAIRO_LT_AGE=17
 
 dnl ===========================================================================
 
-VERSION_INFO="$LT_CURRENT:$LT_REVISION:$LT_AGE"
+VERSION_INFO="$CAIRO_LT_CURRENT:$CAIRO_LT_REVISION:$CAIRO_LT_AGE"
 AC_SUBST(VERSION_INFO)
 
-LT_CURRENT_MINUS_AGE=`expr $LT_CURRENT - $LT_AGE`
-AC_SUBST(LT_CURRENT_MINUS_AGE)
+CAIRO_LT_CURRENT_MINUS_AGE=`expr $CAIRO_LT_CURRENT - $CAIRO_LT_AGE`
+AC_SUBST(CAIRO_LT_CURRENT_MINUS_AGE)
 
 CAIRO_VERSION_MAJOR=cairo_version_major()
 CAIRO_VERSION_MINOR=cairo_version_minor()
@@ -60,6 +60,7 @@
 dnl ===========================================================================
 
 AC_PROG_CC
+AC_PROG_CXX
 AC_PROG_CPP
 AC_LIBTOOL_WIN32_DLL
 AC_PROG_LIBTOOL dnl required version (1.4) DON'T REMOVE!
--- origsrc/cairo-1.6.4/src/Makefile.am	2008-04-11 15:46:05.000000000 -0500
+++ src/cairo-1.6.4/src/Makefile.am	2008-05-13 21:24:56.140625000 -0500
@@ -337,7 +337,7 @@
 	grep -E '^cairo_.* \(' | \
 	sed -e 's/[ 	].*//' | \
 	sort; \
-	echo LIBRARY libcairo-$(LT_CURRENT_MINUS_AGE).dll; \
+	echo LIBRARY libcairo-$(CAIRO_LT_CURRENT_MINUS_AGE).dll; \
 	) >$@
 	@ ! grep -q cairo_ERROR $@ || ($(RM) $@; false)
 
